### PR TITLE
create bare home page

### PR DESCRIPTION
**Pull request by savwiley**

### CODE REVIEW

File: src/components/Home/index.js [hunk 1]
```diff
@@ -10,11 +10,14 @@ const Home = () => {
           Edit <code>src/App.js</code> and save to reload.
         </p>
         <AppLink
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
+          href="URL"
```
Comment (rankoliang) [inline src/components/Home/index.js]: If you want a placeholder, you can use `href="#"`

File: src/components/Home/index.js [hunk 2]
```diff
@@ -10,11 +10,14 @@ const Home = () => {
...
         >
-          Learn React
+          Battle!
+        </AppLink>
+        <AppLink
+          href="URL"
```
Comment (rankoliang) [inline src/components/Home/index.js]: See above comment

File: src/components/Home/styled.js [hunk 3]
```diff
@@ -10,31 +10,20 @@ export const AppHeader = styled.header`
   display: flex;
   flex-direction: column;
   align-items: center;
-  justify-content: center;
   font-size: calc(10px + 2vmin);
-  color: white;
-`;
-
-export const AppLogo = styled.img`
```
Comment (rankoliang) [inline src/components/Home/styled.js:18]: This is deleted, but is still imported in `Home/index.js`.

**Review by rankoliang — COMMENTED** [review]
There were some more things that you changed that were not in this commit. Please re add them back in!

File: src/components/Home/styled.js (hunk 3 above)
Comment (savwiley) [reply src/components/Home/styled.js:18]: Removed the import in Home/index.js

**Comment by savwiley** [comment]
Closing PR. New one is [here](https://github.com/rankoliang/dino-battle/pull/18).

**Pull request closed without merging**